src/app/story/directives/status.component.ts

```diff
@@ -156,6 +158,7 @@ export class StoryStatusComponent implements DoCheck {
   togglePublish() {
     this.isPublishing = true;
     this.story.setPublished(!this.story.publishedAt).subscribe(() => {
+      this.angulartics2.eventTrack.next({ action: this.story.publishedAt ? 'publish' : 'unpublish', properties: { category: 'episode', label: 'episode/' + this.story.doc.id }});
```

Comment:
also, `this.story.publishedAt` is used to determine publish/unpublish 3x in 3 lines -- could save as a local var?